runtime/vm/ContinuationHelpers.cpp

```diff
@@ -292,6 +293,19 @@ walkContinuationStackFrames(J9VMThread *currentThread, J9VMContinuation *continu
 	return rc;
 }
 
+
+jvmtiIterationControl walkContinuationCallBack(J9VMThread *vmThread, J9MM_IterateObjectDescriptor *object, void *userData) {
+	J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, object->object);
+	if (NULL != continuation) {
+		J9StackWalkState localWalkState = {0};
+		/* Reset localWalkState */
+		localWalkState = *(J9StackWalkState*)userData;
+		/* walk live continuation's stack */
```

Comment:
I would also explicitly state: why we are only walking live continuations' stacks?
```suggestion
		/* Walk live continuation's stack */
```

Reply:
> I would also explicitly state: why we are only walking live continuations' stacks?
Still unresolved.

Reply:
non-null J9VMContinuation implies live stack, not sure how explicit you need, I guess I will update the wording to use non-null instead of live

Reply:
> why are we ONLY walking live continuations' stacks?

rephrased: why are we ignoring unmounted continuations?

Reply:
it's probably a requirement of the API that utilizes walkAllStackFrames. By just reading the surrounding the code, i can't figure out the reason for ONLY walking live continuations' stacks and excluding the unmounted continuations.

Reply:
makes sense now. by live, i thought that it referred to continuations that were not running. misinterpreted the terminology. live == initialized J9VMContinuation.